### ISSUE TITLE
[Bug]: after deleting all store folders, MO restart is still not successful

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#5681]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):7108574f2ca9b3c801faa17bcb11372e1d005598
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

we keep seeing "error: internal error: not enough log stores"

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/4965857/196125220-085b2ff3-5386-4b75-b9cc-a2ff257ac9c7.png">

**Comment by volgariver6** [comment]
resolved by #6012

**Comment by aressu1985** [comment]
fixed:1feeb04254c6fdab0ae0061ecc32349136d6ee10

 but still can not restart ,traced by #6284
